.github/workflows/build.yml

```
@@ -0,0 +1,16 @@
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - uses: ./.github/actions/prepare
+      - run: pnpm build
+      - run: node ./lib/index.js
```

[Tooling] How about running the bin on itself as a form of checking that?

```suggestion
      - run: node ./lib/bin/pjv.mjs
```

---

Updated